### DISCUSSION TITLE
Tell me your favorite sports!!

### DISCUSSION THREAD

**Discussion by donguri7** [NAIST-SE/AttractingContributors#1169] [2024 Attraction Contributions]
Please edit my README.md to tell me your favorite sports!
If you like same sports, don't worry about it!
link: https://github.com/donguri7/AttractingContributers_Favorite_sports

**Comment by wangzaixiaoliunai** [comment]
I have created a PR.
Could you please contribute to mine?
https://github.com/wangzaixiaoliunai/AttractingContributors-musical_instruments.git

**Comment by Marsall-k** [comment]
I have just made a PR on your Repo!
could you contribute to mine?
https://github.com/Marsall-k/AttractingContributors-Pokemon

**Comment by marshr00m** [comment]
I submitted a pull request. I would appreciate any additions to my list!
https://github.com/marshr00m/AttractingContributors-Where-would-you-like-to-travel
